features/steps/core.py

```diff
@@ -423,6 +430,7 @@ def _mock_editor(command):
             patch("sys.stdin.read", side_effect=lambda: text), \
             patch("jrnl.time.parse", side_effect=_mock_time_parse(context)), \
             patch("jrnl.config.get_config_path", side_effect=lambda: context.config_path), \
+            patch("jrnl.install.load_or_install_jrnl",wraps=jrnl.install.load_or_install_jrnl), \
```

Comment:
What's the purpose of this? I don't deeply understand how `wraps` works, though I did try removing this line and all tests still passed.

Reply:
hmm... I forgot why I added it in the first place and you're right, it isn't needed. 

As for `wraps`, I've seen (and used) it as a spy to check the live call to a function. This way I can exercise the code and have checks like `assert_called_with`